### PR TITLE
PlaceView supports custom

### PR DESCRIPTION
**Pull request by laurarodgers**
PlaceView supports custom criteria, calculates averages using array rating.

### CODE REVIEW

**Comment by sassansh** [comment]
Such an elegant design! Absolutely love it. 🤩 Fix the lint issues and merge 👍🏻 
image

**Comment by laurarodgers** [comment]
The linter thinks Ant props like flex="0 0" are zeros with units, which is a real irritant

Also not loving how 2.45 rounds to 2, not 2.5 according to Ant Rate, what even is that, do we need to hard round in advance each time or use a custom version of rate

File: client/src/components/RatingCriterion/RatingCriterion.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.crit-name {
+  font-size: large;
+  margin: 5px 0px 0px 0px;
```
Comment (sassansh) [inline client/src/components/RatingCriterion/RatingCriterion.css]:
```
2021-08-05 06:15:47 [INFO]   File:[/github/workspace/client/src/components/RatingCriterion/RatingCriterion.css]
2021-08-05 06:15:48 [ERROR]   Found errors in [stylelint] linter!
2021-08-05 06:15:48 [ERROR]   Error code: 2. Command output:
------

client/src/components/RatingCriterion/RatingCriterion.css
 3:16  ✖  Unexpected unit   length-zero-no-unit
 3:20  ✖  Unexpected unit   length-zero-no-unit
 3:24  ✖  Unexpected unit   length-zero-no-unit
------
```

The `length-zero-no-unit` error is referring to this section on **line 3**. It’s saying that 0 length doesn’t need a unit. So just change the `0px` to `0`.

**Comment by sassansh** [comment]
> Also not loving how 2.45 rounds to 2, not 2.5 according to Ant Rate, what even is that, do we need to hard round in advance each time or use a custom version of rate

~~I know, I found that weird too. But I looked at Amazon. They do it the same way as ant as well. They round down to the closest .5 rating. So it’s pretty standard. It should be ok.~~

Edit: Never mind. They do round up sometimes too:
image
image

**Review by sassansh — APPROVED** [review]
Decided to help out and fix the lint issues :) Let me know if you have any questions! Its ready to be merged ✅

Amazing job! 👏🏻

**Comment by laurarodgers** [comment]
> Decided to help out and fix the lint issues :) Let me know if you have any questions! Its ready to be merged ✅
> 
> Amazing job! 👏🏻

Please don't do this. I don't know how else to phrase it. Please don't push commits to branches that literally have other people's names on them. It makes it very difficult to do my own work on my own branch.

**Pull request merged**